src/sqlfluff/dialects/dialect_postgres.py

```diff
@@ -44,7 +44,16 @@
 
 ansi_dialect = load_raw_dialect("ansi")
 
-postgres_dialect = ansi_dialect.copy_as("postgres")
+postgres_dialect = ansi_dialect.copy_as(
+    "postgres",
+    formatted_name="PostgreSQL",
+    docstring="""This is based around the `PostgreSQL spec`_. Many other SQL
+dialects are often based on the PostreSQL syntax. If you're running an unsupported
+dialect, then this is often the dialect to use (until someone makes a specific
+dialect).
+
+.. _`PostgreSQL spec`: https://www.postgresql.org/docs/9.6/reference.html""",
```

Comment:
This should probably reference the most current docs.
```suggestion
.. _`PostgreSQL spec`: https://www.postgresql.org/docs/current/reference.html""",
```

Reply:
Good idea 👍